scripts/ci-smoketest.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Travis CI Release / Smoketest Script
+#
+LAYER0_PATH=$GOPATH/src/github.com/quintilesims/layer0
+LAYER0_PREFIX=smoketest
+set -e
+set -x
+
+# Ignore if master branch is not target
+if [ ! "$TRAVIS_BRANCH" == "master" ]; then
+  echo "[INFO] Skipping Smoketest";
+  exit 0
+fi
+
+# Tag repository
+git tag "$LAYER0_PREFIX"
```

Comment:
Two reasons: 
1) too many image hashes in our Dockerhub --- do we really need a separate hash for every smoketest?
2) when I was previously running this all in a Docker container, it turns out Travis does not preserve tag data, so our method of `git describe` was return nothing.